database/nakadi/10_data/04_tables/05_timeline.sql

```diff
@@ -8,6 +8,7 @@ CREATE TABLE zn_data.timeline (
   tl_switched_at     TIMESTAMP WITH TIME ZONE DEFAULT NULL,
   tl_cleanup_at      TIMESTAMP WITH TIME ZONE DEFAULT NULL,
   tl_latest_position JSONB                    DEFAULT NULL,
+  tl_deleted         BOOLEAN                  DEFAULT FALSE,
```

Comment:
That's a good idea, but unfortunately the column `st_id` in table `timeline` is a foreign key to `st_id` in table `storage`. So we wouldn't be able to delete a storage, if there is one deleted timeline left on the storage.